experimenter/experimenter/jetstream/tests/constants.py

```diff
@@ -480,7 +480,6 @@ def get_test_data(cls, primary_outcomes):
             EXPOSURES_VARIANT_NEGATIVE_SIGNIFICANCE_DATA_ROW.dict(exclude_none=True),
             EXPOSURES_CONTROL_NEUTRAL_SIGNIFICANCE_DATA_ROW.dict(exclude_none=True),
             EXPOSURES_BROKEN_STATISTIC_DATA_ROW.dict(exclude_none=True),
-            VARIANT_EXPOSURES_BROKEN_STATISTIC_DATA_ROW.dict(exclude_none=True),
```

Comment:
This looked strange to me so I checked -- it's actually a bug in the test data. You can leave this line in there, and then just make one small change up above:
Line 402 currently reads:
```py
VARIANT_EXPOSURES_BROKEN_STATISTIC_DATA_ROW = BROKEN_STATISTIC_DATA_ROW.copy()
```
but it should be:
```py
VARIANT_EXPOSURES_BROKEN_STATISTIC_DATA_ROW = VARIANT_BROKEN_STATISTIC_DATA_ROW.copy()
```

Can you make the change there instead? Sorry about that!

Reply:
Sure I can do that!